README.md

```diff
@@ -214,6 +214,44 @@ PLAY RECAP *********************************************************************
 localhost                  : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
 ```
 
+[This](./playbooks/droplet.yml) is a sample playbook which registers an SSH key and a Droplet which uses it:
+
+```yaml
+---
+- name: Create SSH key and Droplet
+  hosts: localhost
+  connection: local
+  gather_facts: true
+
+  vars:
+    digitalocean_token: "{{ lookup('ansible.builtin.env', 'DIGITALOCEAN_TOKEN') }}"
+    public_key: "{{ lookup('ansible.builtin.file', ansible_env['HOME'] ~ '/.ssh/sammy.key.pub') }}"
+
+  tasks:
+    - name: Create SSH key
+      digitalocean.cloud.ssh_key:
+        state: present
+        token: "{{ digitalocean_token }}"
+        public_key: "{{ public_key }}"
+        name: "sammy"
+      register: ssh_key
+
+    - name: Create Droplet
+      digitalocean.cloud.droplet:
+        state: present
+        token: "{{ digitalocean_token }}"
+        name: sammy-droplet
+        region: nyc3
+        size: s-1vcpu-2gb
+        image: ubuntu-22-04-x64
+        ssh_keys: ["{{ ssh_key.ssh_key.id }}"]
+        unique_name: true
+```
+
+> **Note**
```

Comment:
Technically github is moving to new syntax for these (and also added an `important`): https://github.com/orgs/community/discussions/16925

```suggestion
> [!NOTE]
```

Reply:
Ah, nice, thanks for letting me know.